generators/entity-client/templates/angular/src/main/webapp/app/entities/_entity-management-detail.component.html

```diff
@@ -26,7 +26,7 @@ <h2><span jhiTranslate="<%= keyPrefix %>.detail.title"><%= entityClassHumanized
         const fieldName = fields[idx].fieldName;
         const fieldType = fields[idx].fieldType;
         const fieldTypeBlobContent = fields[idx].fieldTypeBlobContent; _%>
-        <dt><span jhiTranslate="<%= keyPrefix %>.<%= fieldName %>"><%= fields[idx].fieldNameHumanized %></span></dt>
+        <dt><span jhiTranslate="<%= keyPrefix %>.<%= fieldName %>" <%_ if (fields[idx].javadoc) { _%> [ngbTooltip]="'<%= keyPrefix %>.help.<%= fieldName %>' | translate" <%_ } _%>><%= fields[idx].fieldNameHumanized %></span></dt>
```

Comment:
what about when i18n is disabled?

Reply:
use `<% %>` and not `<%_ _%>` as the first one is for inline and second one is for newline. Same for all other inline usage <%_ should never be used inline

Reply:
ok i will do it.

Reply:
Good question for i18n disabled. I have no idea how jhipster works in this case.
Is there any flag to detect it ? And in this case i put directly the javadoc value, ok ?